tests/test_D1_model_components.py

```diff
@@ -718,6 +730,145 @@ def test_storage_fix(self):
+        dict_asset = self.dict_values[ENERGY_STORAGE]["storage_optimize"]
+        dict_asset[STORAGE_CAPACITY][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[INPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[OUTPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL] = {VALUE: 0, UNIT: "no_unit"}
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS] = {VALUE: 0.001, UNIT: "kWh"}
+
+        D1.storage(
+            model=self.model,
+            dict_asset=dict_asset,
+            bus=self.busses,
+            storage=self.storages,
+        )
+
+        assert (
+            self.model.entities[-1].investment.minimum == 1
+        ), f"investment.minimum should be one with non-zero {THERM_LOSSES_REL} and {THERM_LOSSES_ABS}"
+
+    def test_storage_optimize_investment_minimum_1_rel_times_series(self):
+        # Test if minimum value is one if relative thermal losses are not zero (time series)
+        dict_asset = self.dict_values[ENERGY_STORAGE]["storage_optimize"]
+        dict_asset[STORAGE_CAPACITY][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[INPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[OUTPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL] = {
+            VALUE: [0.001, 0.001, 0.001, 0.001],
+            UNIT: "no_unit",
+        }
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS] = {VALUE: 0, UNIT: "kWh"}
+
+        D1.storage(
+            model=self.model,
+            dict_asset=dict_asset,
+            bus=self.busses,
+            storage=self.storages,
+        )
+
+        assert (
+            self.model.entities[-1].investment.minimum == 1
+        ), f"investment.minimum should be one with non-zero {THERM_LOSSES_REL} and {THERM_LOSSES_ABS}"
+
+    def test_storage_optimize_investment_minimum_1_abs_times_series(self):
+        # Test if minimum value is one if absolute thermal losses are not zero (time series)
+        dict_asset = self.dict_values[ENERGY_STORAGE]["storage_optimize"]
+        dict_asset[STORAGE_CAPACITY][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[INPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[OUTPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL] = {VALUE: 0, UNIT: "no_unit"}
+        dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS] = {
+            VALUE: [0.001, 0.001, 0.001, 0.001],
+            UNIT: "kWh",
+        }
+
+        D1.storage(
+            model=self.model,
+            dict_asset=dict_asset,
+            bus=self.busses,
+            storage=self.storages,
+        )
+
+        assert (
+            self.model.entities[-1].investment.minimum == 1
+        ), f"investment.minimum should be one with non-zero {THERM_LOSSES_REL} and {THERM_LOSSES_ABS}"
```

Comment:
```suggestion
        ), f"investment.minimum should be one with {THERM_LOSSES_REL} and non-zero {THERM_LOSSES_ABS}"
```

Reply:
Perfect, I completely overlooked this!
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/6005fc7a88893ea15c0bccb70d8c5bd596c8fe5c